libs/vector-map/vector-map.cabal

```diff
@@ -33,10 +33,28 @@ library
         deepseq,
         nothunks,
         primitive,
-        tree-diff,
         vector,
         vector-algorithms
 
+library testlib
```

Comment:
Addition of testlib in this package, is in line with how we do things everywhere in ledger, so this is also how it should have been done for all other packages